README.md

```diff
@@ -1,34 +1,60 @@
-# Laserfiche Repository Client API Sample - Service App
-Sample project for demonstrating how to work with our Repository API client.
+# lf-sample-repository-api-dotnet-srv
 
-## Steps to recreate this projecct
+Sample .NET core service app that connects to a Laserfiche Cloud Repository using a service principal account.
+[Sample Code](./Program.cs)
 
-### Scafolding
+### Prerequisites
 
-```
-dotnet new console --framework net6.0
-```
+#### Software Prerequisites
 
-### Run the project
+- Visual Studio Code
+- .NET 6.0 core SDK
+- Git
+- Clone this repo on your local machine
 
-```
-dotnet run
-```
+#### 1. Create a Service Principal
 
-### Add dependencies
+- Login to account using web client as an administrator
+- Using the app picker, go to the account page
+- Click on the service principal tab
+- Click on the 'Add Service Principal' button to create an account to be used to run this sample service
```

Comment:
Shall we unify the way we call these UI element? Like: 'click on "Name of the Button"' or 'click on "Name of the Tab"' instead of 'click on "Name of the Button" button' or 'click on "Name of the Tab" tab'.

Reply:
OK, as long as it's consistent.